sound/soc/sof/intel/pci-tgl.c

```diff
@@ -61,6 +62,7 @@ static const struct sof_dev_desc tglh_desc = {
 	.chip_info = &tglh_chip_info,
 	.ipc_supported_mask	= BIT(SOF_IPC) | BIT(SOF_INTEL_IPC4),
 	.ipc_default		= SOF_IPC,
+	.dspless_mode_supported	= true,		/* Only supported for HDaudio */
 	.default_fw_path = {
 		[SOF_IPC] = "intel/sof",
```

Comment:
in dspless mode,  which topology will be used ?

Reply:
But yes, we are loading the topology file to get the DAIs for the PCMs.